system-test/tedious-connect.cjs

```diff
@@ -0,0 +1,80 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const t = require('tap');
+const semver = require('semver');
+const {Connector} = require('@google-cloud/cloud-sql-connector');
+
+t.test(
+  'open connection and run basic sqlserver commands',
+  {skip: semver.lt(process.versions.node, '16.0.0')},
+  async t => {
+    // lazy-load tedious in order to skip node14
```

Comment:
as far as I'm aware this is the first time we're skipping tests in node14 - maybe you're mixing it with the instance in which node14 is missing a key API and we have to add an extra function just to make parsing certificates work? see: https://github.com/GoogleCloudPlatform/cloud-sql-nodejs-connector/blob/7aa2057eeb70cfd355d381524702404a309e2bbb/src/crypto.ts#L21-L67

Reply:
tbh there's not much to be done here other than skipping tests. the issue is that `tedious` itself is not compatible with node14 since they're using syntax that is only available in node16+

Reply:
Oh in that case, let's just make that clear: Tedious doesn't support node14, so skip the test.

Reply:
sounds good! 👍